libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.ts

```diff
@@ -158,19 +171,34 @@ export class SkyAgGridRowDeleteDirective
   #elementRef: ElementRef;
   #overlayService: SkyOverlayService;
   #viewContainerRef: ViewContainerRef;
+  #scrollableHostService: SkyScrollableHostService;
+  #clipPath$ = new BehaviorSubject<string | undefined>(undefined);
+  #zIndex = new BehaviorSubject('998');
 
   constructor(
     affixService: SkyAffixService,
     changeDetector: ChangeDetectorRef,
     elementRef: ElementRef,
     overlayService: SkyOverlayService,
-    viewContainerRef: ViewContainerRef
+    viewContainerRef: ViewContainerRef,
+    scrollableHostService: SkyScrollableHostService,
+    @Optional()
+    @Inject(SKY_STACKING_CONTEXT)
+    stackingContext?: SkyStackingContext | undefined
   ) {
     this.#affixService = affixService;
     this.#changeDetector = changeDetector;
     this.#elementRef = elementRef;
     this.#overlayService = overlayService;
     this.#viewContainerRef = viewContainerRef;
+    this.#scrollableHostService = scrollableHostService;
+    if (stackingContext) {
+      stackingContext.zIndex
+        .pipe(takeUntil(this.#ngUnsubscribe))
+        .subscribe((zIndex) => {
+          this.#zIndex.next(zIndex.toString(10));
```

Comment:
Possible to use `number` for the type of `#zIndex` to avoid the conversion?

Reply:
Switched to number, converting to string when setting the overlay property.